soh/soh/GameMenuBar.cpp

```diff
@@ -786,6 +790,18 @@ namespace GameMenuBar {
                     UIWidgets::Tooltip("Stops the game from freezing the player when picking up Gold Skulltulas");
                     UIWidgets::PaddedEnhancementCheckbox("Fast Chests", "gFastChests", true, false);
                     UIWidgets::Tooltip("Kick open every chest");
+                    UIWidgets::PaddedText("Chest size & texture matches contents", true, false);
+                    const char* chestSizeAndTextureMatchesContentsOptions[4] = { "Disabled", "Both", "Texture Only", "Size Only"};
+                    UIWidgets::EnhancementCombobox("gChestSizeAndTextureMatchesContents", chestSizeAndTextureMatchesContentsOptions, 4, 0);
+                    UIWidgets::Tooltip(
+                        "Chest sizes and textures are changed to help identify the item inside.\n"
+                        " - Major items: Large gold chests\n"
+                        " - Lesser items: Large brown chests\n"
+                        " - Junk items: Small brown chests\n"
+                        " - Small keys: Small silver chest\n"
+                        " - Boss keys: Vanilla size and texture\n"
+                        " - Skulltula Tokens: Small skulltula chest\n"
```

Comment:
seems this was relying on both size and texture to communicate item category, now that we can do one or the other, it might be cool to make it so all categories are able to be communicated by texture alone.

I'm really not sure what to put in the tooltip with all the new options, maybe just dropping the category descriptions entirely?

Reply:
yeah, I think the description works fine still as it lists both the texture and size, with the exception of Lesser + junk looking the same for size only, not really sure how to word it better for that case

Reply:
yeah, now that i think of it lesser and junk being the same when size doesn't match isn't an issue.